python/cugraph/dask/traversal/mg_bfs.pxd

```diff
@@ -18,11 +18,12 @@ from cugraph.structure.graph_primtypes cimport *
 from libcpp cimport bool
 
 
-cdef extern from "algorithms.hpp" namespace "cugraph":
+cdef extern from "utilities/cython.hpp" namespace "cugraph::cython":
 
-    cdef void bfs[VT,ET,WT](
+    cdef void call_bfs[VT,WT](
```

Comment:
We'd better replace VT, ET, WT with vertex_t, edge_t, weight_t.

Reply:
updated